ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -2929,3 +2929,94 @@ def get_csi_images_for_client_ocp_version(ocp_version=None):
     csi_ocp_version_images = csi_images.split(first_str)[1].split(last_str)[0]
     csi_ocp_version_images_urls = extract_image_urls(csi_ocp_version_images)
     return csi_ocp_version_images_urls
+
+
+def parse_ceph_osd_df_tree_weight_and_size():
```

Comment:
this has similarity to https://github.com/red-hat-storage/ocs-ci/pull/10359/files#diff-a4d73d82c94d13634aded838ea2490c117996218f405a8da15f9ecea093f3f6dR1331

Would it be possible to use the same function for both cases?

Reply:
The values are not the same, but changing the `header` value may work in our case. I will check it.